torsions/TorsionScanSet.py

```diff
@@ -203,7 +203,7 @@ def compute_energy(self, param, platform=None):
             energy = state.getPotentialEnergy()._value
             mm_energy.append(energy)
             #print "frame %5d / %5d : %8.3f kJ/mol" % (i, self.n_frames, energy) # DEBUG
-            delta_energy.append(self.qm_energy[i]._value - energy)
+            delta_energy.append(self.qm_energy[i]._value - energy + offset)
```

Comment:
I think it has to be uniform.  

@ChayaSt is correct that it doesn't matter if it is added or subtracted as long as the range of the uniform distribution is big enough that we end up with
```
(QM-QM_zero) - (MM-MM_zero) = QM - MM + (-QM_zero - MM_zero)
```